src/main/com/example004/radek/DrawMeSquareApplication.java

```diff
@@ -0,0 +1,51 @@
+package main.com.example004.radek;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class DrawMeSquareApplication {
+
+    public static void main(String[] args) {
+
+        int size = getEnteredValue("How big square you wanna draw ? ");
+        String symbol = getEnteredValue();
+        drawMeSquare(size, symbol);
+    }
+
+    static void drawMeSquare(int size, String symbol) {
+
+        for (int row = 0; row < size; row++) {
+            for (int column = 0; column < size; column++) {
+                System.out.print((isTrue(row, size) || isTrue(column, size)) ? symbol : " ");
+            }
+            System.out.println();
+        }
+    }
+
+    static boolean isTrue(int row, int size) {
```

Comment:
přejmenovat název metody ... pokud metoda vrací boolean, tak by měla začínat vždy is, has, contains ... a tím se určí, že se ptáš na boolean a to true je tedy z kontextu is, true, contains ... jasné a hlavně nic neříkající ... pokud bych tu metodu chtěl použít, tak podle názvu vůbec nvm co to dělá ...

Reply:
Asi trochu nerozumím, metoda začíná "is", takže má začínat vždy "is", "has", "contains", nebo nemá ? Nebo upřesnění co je pravda ?

Reply:
is je v poradku, ale kazda metoda isNECO je otazka na isPravda ....
Kdyz budu tu metodu chtit pouzit nekde jinde ... vse by melo byt jasne z nazvu a parametru ... co mi takove isTrue rekne? jePravda? .. to muze byt uplne cokoliv ... ale kdyby tam bylo treba isSmall? isBig? to uz smysl dava

Reply:
Přejmenováno na isPerimeter, jelikož kontroluje, zda se jedná o obvod

Reply:
stále to není ideál

Reply:
Přejmenoval jsem proměnnou na position, aby bylo jasnější co se posílá a metoda je teď isPositionPerimeter

Reply:
aniz bych musel nad tim nejak badat, tak hned to dava smysl tedka